### DISCUSSION TITLE
TuneIn favorite stations

### DISCUSSION THREAD

**Discussion by omakoti** [music-assistant/support#2240] [Q&A] [answered]
Hi.
Just wanted to ask if anyone else is havin the same issue in MA when adding radio stations to TuneIn.
I added 5 stations to favorites in my TuneIn account but only 2 stations show up in MA.
What could be the reason for this? I have reloaded  everything and added the station again but it does not help.

**Comment by OzGav** [comment]
Are the stations that are not appearing PREMIUM?

edit: If so https://github.com/orgs/music-assistant/discussions/1190#discussioncomment-5755777

**Reply by omakoti** [reply]
No, they are not. I don't have the premium and I can listen to those channels inside tunein but they just won't show up in MA

**Comment by omakoti** [comment]
This is one of the stations not showing in MA: https://tunein.com/radio/Radio-Rock-949-s84138/

**Reply by OzGav** [reply]
I just added that station to my favourites as well as a station near me. Interestingly neither station appeared when I did a `synchronise now` so might be a bug there but I restarted the addon and the one near me appeared. So in the past with these issues the final causes included DRM/Copyright problems outside of the TuneIn app/web player. So try a restart of the addon and see if any more appear. If not choose some random station from other countries and see if they work ok. If so best guess right now is DRM but we can look at it further when time allows but currently workload is high

**Reply by omakoti** [reply]
I added 9 stations from my country and 1 station from another country and restarted the addon.
MA now shows 4 stations from my country and also that 1 station from another country. So it indeed seems that the problem is with those few stations and not the addon itself. Hope you can solve this with time. :)

**Reply by OzGav** [reply]
Ask about this again in a couple of months once things settle down.

**Reply by OzGav** [reply, marked as answer]
I was just able to add this station successfully
